### PR TITLE
Update data.ts to include metrics for depth estimation

### PR DESCRIPTION
**Pull request by balugans**
Update data.ts to include metrics for depth estimation. Reference: https://openaccess.thecvf.com/content_CVPR_2019/papers/Lee_Monocular_Depth_Estimation_Using_Relative_Depth_Maps_CVPR_2019_paper.pdf (Page 6)

### CODE REVIEW

File: tasks/src/depth-estimation/data.ts [hunk 1]
```diff
@@ -21,7 +21,25 @@ const taskData: TaskDataCustom = {
 			},
 		],
 	},
-	metrics: [],
+	metrics: [
+		// Reference: https://openaccess.thecvf.com/content_CVPR_2019/papers/Lee_Monocular_Depth_Estimation_Using_Relative_Depth_Maps_CVPR_2019_paper.pdf
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/data.ts:25]: You can just remove this part as the code is already documentation

File: tasks/src/depth-estimation/data.ts [hunk 2]
```diff
@@ -21,7 +21,25 @@ const taskData: TaskDataCustom = {
...
+		{
+			description: "Estimate Root Mean Sqaure Error (RMSE) between depth-estimate and ground-truth",
```
Comment (merveenoyan) [inline tasks/src/depth-estimation/data.ts:27]:
```suggestion
			description: "Estimate Root Mean Square Error (RMSE) between the depth estimate and the ground truth.",
```

**Review by merveenoyan — COMMENTED** [review]
Thanks a lot for this PR!

File: tasks/src/depth-estimation/data.ts [hunk 3]
```diff
@@ -21,7 +21,25 @@ const taskData: TaskDataCustom = {
...
+		{
+			description: "Estimate Root Mean Sqaure Error (RMSE) between logarithmic depth-estimate and logarithmic ground-truth",
+			id: "RMSE_Log",
+		},
+		{
+			description: "Estimate Absolute Relative distance (ARD) between depth-estimate and ground-truth",
+			id: "ARD",
+		},
+		{
+			description: "Estimate Squared Relative distance (ARD) between depth-estimate and ground-truth",
```
[the remark this answers is not in the document]
Comment (balugans) [reply tasks/src/depth-estimation/data.ts:39]: Hi @merveenoyan thanks for the feedback. For all the parameters there are formula available and easier to display using Latex. Can I display equations in typescript?

[the remark this answers is not in the document]
Comment (merveenoyan) [reply tasks/src/depth-estimation/data.ts:39]: I think there's no need for this, plain English is fine if possible :) if not, you can leave them as is.

**Comment by merveenoyan** [comment]
Hey @balugans,
We have moved the codes for Tasks project to [huggingface.js repository](https://github.com/huggingface/huggingface.js) so if you feel like continuing this pull request you can move your work over there. Many thanks! 🤗

**Comment by balugans** [comment]
so sorry I thought i had finished this task. I will move it across.

**Pull request closed without merging**